content/influxdb/cloud/reference/flux/stdlib/built-in/inputs/from.md

```diff
@@ -39,14 +39,18 @@ from(
 )
 ```
 
-## Parameters
-
 {{% note %}}
-[host](#host), [org](#org) or [orgID](#orgid), and [token](#token) parameters
-are only required when querying data from a **different organization** or a
-**remote InfluxDB instance**.
+#### Query remote InfluxDB data sources
+`from()` can retrieve data from remote InfluxDB 2.0 OSS and InfluxDB Cloud instances.
+To query remote InfluxDB sources, include the [host](#host), [token](#token), and
+[org](#org) or [orgID](#orgid) parameters.
+
+`from()` **cannot** retrieve data from **remote InfluxDB OSS 1.x instances or remote InfluxDB Enterprise 1.x clusters**
```

Comment:
```suggestion
`from()` **cannot retrieve data** from **remote** InfluxDB OSS 1.x or InfluxDB Enterprise 1.x data sources
```

Reply:
does `from()` work locally w 1.x sources?
Or would it be accurate to say `from()` isn't supported with 1.x?

Reply:
Yes, it does. InfluxDB 1.7+ and InfluxDB Enterprise 1.7.3+ ship with Flux. The only caveat is that Flux can only query the local 1.x storage engine or a remote 2.0 instance.